src/StackedTableRow.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import css from 'styled-jsx/css'
+import propTypes from 'prop-types'
+import { instanceOfComponent } from '@dhis2/prop-types'
+
+import { StackedTableCell } from './StackedTableCell.js'
+import { StackedTableCellHead } from './StackedTableCellHead.js'
+
+const addColNumToChildren = children => {
+    let curCol = 0
+
+    return React.Children.map(children, child => {
+        const column = curCol
+        const colSpan = child.props.colSpan
+            ? parseInt(child.props.colSpan, 10)
+            : 1
+
+        curCol += colSpan
+
+        return React.cloneElement(child, { column })
+    })
+}
+
+const tableRowStyles = css`
+    tr {
+        min-height: 45px;
+        display: block;
+        border: 1px solid #e8edf2;
+    }
+
+    tr:nth-child(even) {
+        background: none;
+    }
+
+    :global(thead) tr,
+    :global(tbody) tr {
+        min-height: 36px;
+    }
+
+    tr + tr {
+        margin-top: 32px;
+    }
+
+    tr:nth-child(even) :global(td) {
+        background: #fbfcfd;
```

Comment:
I think we should turn off the zebra-striping for these stacked cells. They are separated by space and borders, so I think a change in background color would only be confusing.